### DISCUSSION TITLE
How should sklearn n_jobs parameter be used with this library?

### DISCUSSION THREAD

**Discussion by smith558** [intel/scikit-learn-intelex#1164] [General]
How should the `n_jobs` parameter (https://scikit-learn.org/stable/glossary.html#term-n_jobs) present in many scikit-learn estimators be used with this library? What's the most optimal use?

**Comment by Alexsandruss** [comment]
Currently, sklearnex ignores `n_jobs`. This parameter is used onIy if algorithm fallbacks to default sklearn implementation.
Number of threads used by sklearnex can be regulated with `daal4py.daalinit` function:
```python
import daal4py as d4p
d4p.daalinit(n_threads)
```

**Comment by napetrov** [comment]
and you can find full examples here for controlling threads - https://github.com/intel/scikit-learn-intelex/blob/master/examples/sklearnex/n_jobs.py
